app/controllers/users_controller.rb

```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find(params[:user_id])
+  end
+end
```

Comment:
- [ ] It is required to have index and show actions in your usersController according to the project requirement.

### All the users (index)
![all-users](https://user-images.githubusercontent.com/26597432/236910905-21cbc2b2-e6c9-426c-a16e-bd796d9b4ad9.png)

### show action
![single-user](https://user-images.githubusercontent.com/26597432/236910979-72f2ddb7-8245-4d5b-b71e-04af13ec699c.png)